src/compas_rhino/geometry/surfaces/surface.py

```diff
@@ -168,22 +168,27 @@ def from_rhino(cls, rhino_surface):
         return curve
 
     @classmethod
-    def from_plane(cls, plane, box):
+    def from_plane(cls, plane, box=None):
         """Construct a surface from a plane.
 
         Parameters
         ----------
         plane : :class:`compas.geometry.Plane`
             The plane.
+        box : :class:`compas.geometry.Box`, optional
+            A box that bounds the surface.
 
         Returns
         -------
         :class:`compas_rhino.geometry.RhinoSurface`
 
         """
         plane = plane_to_rhino(plane)
-        box = Rhino.Geometry.BoundingBox(box.xmin, box.ymin, box.zmin, box.xmax, box.ymax, box.zmax)
-        rhino_surface = Rhino.Geometry.PlaneSurface.CreateThroughBox(plane, box)
+        if box:
+            box = Rhino.Geometry.BoundingBox(box.xmin, box.ymin, box.zmin, box.xmax, box.ymax, box.zmax)
+            rhino_surface = Rhino.Geometry.PlaneSurface.CreateThroughBox(plane, box)
+        else:
+            rhino_surface = Rhino.Geometry.PlaneSurface(plane, Rhino.Geometry.Interval(0, 1), Rhino.Geometry.Interval(0, 1))
```

Comment:
also in this case, it would be helpful if the user could set this. would be the alternative to providing a bounding box...